src/fixtures/stubbed_saved_object_index_pattern.ts

```diff
@@ -33,8 +33,8 @@ import stubbedLogstashFields from './logstash_fields';
 
 const mockLogstashFields = stubbedLogstashFields();
 
-export function stubbedSavedObjectIndexPattern(id: string | null = null) {
-  return {
+export function stubbedSavedObjectIndexPattern(id: string | null = null, withDataSource?: boolean) {
```

Comment:
give `withDataSource` default value to false?

Reply:
sure,to be more explicit